client/js/apiService.js

```diff
@@ -0,0 +1,47 @@
+const MOVIE_NOT_FOUND = 'Movie not found!';
+
+const extractRottenTomatoRating = (ratingsArray) => {
+    if (ratingsArray.length > 0) {
+        const rottenTomatoRating = ratingsArray[1]
```

Comment:
This is considering some JSON structure, right? 
Let's come up with one which is scalable and can be used easily. If you have one, let us know

Reply:
Backend team will handle this. I have given comments to other PR so that they can return POJO to client